### PR TITLE
Upgrade DOCS 

### PR DESCRIPTION
**Pull request by Ravivarshney**
NOTE: We love contributions, but we hate spam. If your PR doesn't improve flutter.dev, we'll close it and label it as `invalid`.

Fixes #<issue number>.

Changes proposed in this pull request:

*  
*

### CODE REVIEW

**Comment by micpap25** [comment]
@sfshaza2 just want to note that, while this IS obvious Hacktoberfest spam and the second install link is pointless, I think there might be value in linking to the release notes in the readme. This might be useful with some retooling. 
@Ravivarshney if you can revert the second commit, remove the word "out" from the first commit, and properly format the link, then I think this can be accepted. Otherwise I could submit a similar PR.

File: README.md [hunk 1]
```diff
@@ -17,9 +17,14 @@ If you want/need to build, read on.
 ## Before you build this site
 
 ### 1. Get the prerequisites
+Flutter 1.12.13 - Latest Version 
+
+Read out the Flutter 1.12.13 [release notes](https://flutter.dev/docs/development/tools/sdk/release-notes/release-notes-1.12.13)
```
Comment (micpap25) [inline README.md:22]: Really useful, actually. Just remove the word "out".

File: README.md [hunk 2]
```diff
@@ -17,9 +17,14 @@ If you want/need to build, read on.
 ## Before you build this site
 
 ### 1. Get the prerequisites
+Flutter 1.12.13 - Latest Version
```
Comment (micpap25) [inline README.md:20]: Is this something we can reasonably maintain? We could just do "Flutter 1.12" instead.

File: README.md [hunk 3]
```diff
@@ -17,9 +17,14 @@ If you want/need to build, read on.
 ## Before you build this site
 
 ### 1. Get the prerequisites
+Flutter 1.12.13 - Latest Version 
+
+Read out the Flutter 1.12.13 [release notes](https://flutter.dev/docs/development/tools/sdk/release-notes/release-notes-1.12.13)
 
 Install the following tools if you don't have them already.
 
+[Direct install From Flutter officals site](https://flutter.dev/docs/get-started/install)
```
Comment (micpap25) [inline README.md:26]: This is redundant with a link less than 10 lines below this.

**Review by micpap25 — COMMENTED** [review]
Just some comments if @Ravivarshney wants to finish this.

**Review by sfshaza2 — COMMENTED** [review]
Closing.

**Comment by sfshaza2** [comment]
@micpap25, we are getting spammed like crazy from Hacktoberfest contributors and are taking a hard line.

**Comment by micpap25** [comment]
Makes sense

**Pull request closed without merging**